schematics/README.md

```diff
@@ -0,0 +1,49 @@
+# Ui-jar Schematics
```

Comment:
nitpik: Would be good add some reference of this on the main README.md, so it don't get lost :D